### DISCUSSION TITLE
Représentation catégorisée des données via FLUX WFS

### DISCUSSION THREAD

**Discussion by IGNF-Xavier** [IGNF-Ma-carte/.github#1] [Visualiseur] [answered]
La DREAL Normandie ( identifiant github : jpotel) a testé l'intégration des flux WFS et WMS configurés au travers du swagger de l'API Entrepôt.

L'utilisateur indique ne pas avoir trouvé comment faire une représentation catégorisée des données via le flux WFS

**Comment by Viglino** [comment, marked as answer]
L'outil de représentation paramétrique (en fonction d'attributs de la couche) ne sera disponible que dans la prochaine version de l'outil (actuellement en cours de déploiement).
![image](https://github.com/IGNF-Ma-carte/.github/assets/7868217/d4c482af-704c-43f9-9698-d7b045530515)
